server/middleware/reactApplication/index.js

```diff
@@ -62,19 +58,21 @@ export default function reactApplicationMiddleware(request, response) {
     </AsyncComponentProvider>
   );
 
+  const App = () => app;
```

Comment:
place this below asyncBootstrapper, because async bootstrapper run all function to preloaded data in with `react-async-component`, on line 68 that you deleted add this code instead
`const ResolvedApp = () => app;`